### PR TITLE
Add missing `weight` key in node affinity example

### PR DESCRIPTION
**Pull request by dhirschfeld**
**Reason for Change**:

**Requirements**

- [ ] squashed commits
- [ ] included documentation
- [ ] added unit tests and e2e tests (if applicable). See [test standard](https://github.com/Azure/aad-pod-identity/blob/master/CONTRIBUTING.md#test-standard) for more details.
- [ ] ran `make precommit`

**Issue Fixed**:

**Please answer the following questions with yes/no**:

Does this change contain code from or inspired by another project? If so, did you notify the maintainers and provide attribution?

- [X] It doesn't contain code from or inspired by another project.

**Notes for Reviewers**:

### CODE REVIEW

**Comment by dhirschfeld** [comment]
Since there were some existing useful links to k8s docs I took the liberty of adding a couple more which helped me figure out the correct config.

File: manifest_staging/charts/aad-pod-identity/values.yaml [hunk 1]
```diff
@@ -72,10 +73,12 @@ mic:
...
     #   operator: "Exists"
 
+  # ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity
   affinity: {}
     # nodeAffinity:
     #   preferredDuringSchedulingIgnoredDuringExecution:
-    #     - preference:
+    #     - weight 1
+    #       preference:
     #       matchExpressions:
```
Comment (aramase) [inline manifest_staging/charts/aad-pod-identity/values.yaml:82]: The indentation needs to be fixed here:

```yaml
      - weight: 1
        preference:
          matchExpressions:
          - key: kubernetes.azure.com/mode
            operator: In
            values:
            - system
```

File: manifest_staging/charts/aad-pod-identity/values.yaml [hunk 2]
```diff
@@ -180,10 +184,12 @@ nmi:
     # - key: "CriticalAddonsOnly"
     #   operator: "Exists"
 
+  # ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity
   affinity: {}
     # nodeAffinity:
     #   preferredDuringSchedulingIgnoredDuringExecution:
-    #     - preference:
```
Comment (aramase) [inline manifest_staging/charts/aad-pod-identity/values.yaml:186]: The indentation needs to be fixed here:

```yaml
      - weight: 1
        preference:
          matchExpressions:
          - key: kubernetes.azure.com/mode
            operator: In
            values:
            - system
```

Comment (dhirschfeld) [reply manifest_staging/charts/aad-pod-identity/values.yaml:186]: Sorry - copy/paste error! I think 890e7e4cc03560ad0e766df8d22a3ba0153b734f should fix it?

**Review by aramase — APPROVED** [review]
Thank you for the PR, @dhirschfeld!

lgtm

**Pull request merged**